docs/getting_started/troubleshooting.rst

```diff
@@ -193,6 +193,70 @@ Although this example may work on your machine, we do not recommend it, because
 the Python multiprocessing library will duplicate Ray clusters, causing both
 excessive resource usage and conflict over the available resources.
 
+Poor performance of the first operation with Modin on Ray engine
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+There might be cases when the first operation with Modin on Ray engine is much slower than the subsequent calls of the operation.
+That happens because Ray workers may not be fully set up yet to perform computation after initialization of the engine
+with ``ray.init(runtime_env={'env_vars': {'__MODIN_AUTOIMPORT_PANDAS__': '1'}})``.
```

Comment:
I would mention that this is currently the default behaviour on Ray (if Ray wasn't initialized explicitly), and maybe add a link to github PR or add an explanation on why we do this...

Reply:
done